tox.ini

```diff
@@ -18,9 +18,7 @@ changedir = docs
 commands = sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
 
 [testenv:lint]
-deps =
```

Comment:
Eventually, we're going to extend mypy to check interfaces for upstream packages that have type specifications for their APIs. To do this, those packages will need to be installed. In the short term, I'm not sure there's a significant impact for us to installing all of mitmproxy here?